Pod/Classes/CGPoint+KDTreePoint.swift

```diff
@@ -6,7 +6,11 @@
 //
 //
 
-import UIKit
+#if os(OSX)
```

Comment:
No need for this actually. You can just import Foundation and you'll get access to CGPoint on both platforms.